.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -85,25 +119,14 @@ RUN python -m pip install src/common
 RUN python -m pip install --require-hashes -r src/library_generation/requirements.txt
 RUN python -m pip install src/library_generation
 
-# Install nvm with node and npm
```

Comment:
Why Node is not needed anymore? I thought we still need it for running owlbot cli?

Reply:
Oh, that's because Alpine can install `nodejs` + `npm` via `apk`

https://github.com/googleapis/sdk-platform-java/blob/10b176b41b38aac4e8e0469324d4fb102027f535/.cloudbuild/library_generation/library_generation.Dockerfile#L65-L66